plugins/CoreHome/templates/_menu.twig

```diff
@@ -45,15 +45,13 @@
                 {% endfor %}
 
                 {% if hasSubmenuItem %}
-                    <li id="{% if level2._url is defined and level2._url is not empty %}{{ _self.getId(level2._url) }}{% endif %}" class="menuTab">
-
+                    <li role="menuitem" class="menuTab" title="{{ level1|translate|e('html_attr') }}" id="{% if level2._url is defined and level2._url is not empty %}{{ _self.getId(level2._url) }}{% endif %}">
                         <a class="item" href="">
-                            <span class="menu-icon {{ level2._icon|default('icon-arrow-right') }}"></span>{{ level1|translate }}
-                            <span class="hidden">
-                             {{ 'CoreHome_Menu'|translate }}
-                           </span>
+                            <span class="menu-icon {{ level2._icon|default('icon-arrow-right') }}"></span>
+                            {{ level1|translate }}
```

Comment:
No problem for bothering, I'm aware of my situation, learning by mistakes :mortar_board: 
Thank you for the link about squashing commits, I was unaware because not mentioned in Docs. (I've added a PR also on it)